expression/distsql_builtin.go

```diff
@@ -1038,6 +1038,19 @@ func newDistSQLFunctionBySig(sc *stmtctx.StatementContext, sigCode tipb.ScalarFu
 	}, nil
 }
 
+// PBToExprs converts pb structures to expressions.
+func PBToExprs(pbExprs []*tipb.Expr, fieldTps []*types.FieldType, sc *stmtctx.StatementContext) ([]Expression, error) {
+	exprs := make([]Expression, 0, len(pbExprs))
+	for _, expr := range pbExprs {
+		e, err := PBToExpr(expr, fieldTps, sc)
```

Comment:
The `PDToExpr` may return both `e` and `err` nil, it's better to handle this case.

Reply:
Great.